spec/integration/urls_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe 'Permalinks work' do
```

Comment:
How do you feel about this test (that actually hits https://knapsackpro.com)?

Reply:
My take is that the redirect behavior should be tested on the API, not in the library. Same way we don't test other API endpoints from the library. IMO the API sets the standard (permalinks), and the library should just follow it. In essence, I view the permalinks as an internal API of sorts.